infra/lib/ci-cluster.ts

```diff
@@ -49,17 +49,24 @@ export class CICluster extends cdk.Construct {
   }
 
   installFlux = () => {
-    const fluxChart = this.testCluster.addCdk8sChart('flux',
-      new FluxChart(
-        this.cdk8sApp, 'Flux', {}
-      )
-    );
+    const fluxNamespace = this.createNamespace(FLUX_NAMESPACE);
+    const fluxChart = this.testCluster.addHelmChart('flux2', {
+      chart: 'flux2',
+      repository: 'https://fluxcd-community.github.io/helm-charts',
+      namespace: FLUX_NAMESPACE,
+      version: '0.19.1',
```

Comment:
maybe should be 0.19.2? https://github.com/fluxcd-community/helm-charts/blob/7fe7d95627b21a751ccd52725fe1a187cbada1ff/charts/flux2/Chart.yaml#L5

Reply:
Thanks! I will change it.
Updating it may potentially take out all prow components again, so I will set out some time tomorrow(Friday) to do that.

Reply:
Done.